dvc/main.py

```diff
@@ -60,7 +62,13 @@ def main(argv=None):
     except Exception as exc:  # pylint: disable=broad-except
         if isinstance(exc, OSError) and exc.errno == errno.EMFILE:
             logger.exception(
-                "too many open files, please increase your `ulimit`"
+                "too many open files, please visit "
+                "{} to see how to handle this problem".format(
+                    format_link(
+                        "https://dvc.org/doc/user-guide/troubleshooting#"
+                        "too-many-open-files-error"
+                    )
```

Comment:
From https://github.com/iterative/dvc.org/pull/875#discussion_r362920851:
> we need to come up with our own links similar to man.dvc.org. https://dvc.org/doc/user-guide/troubleshooting#too-many-open-files-error looks ugly and a burden for us to support
> Let's discuss and create our codes and URLs like https://error.dvc.org/dvc0001 or https://error.dvc.org/too-many-open-files?

– @shcheklein 

I like Ivan's idea. You could use error.dvc.org/001 (or whatever you decided regarding error codes) here and open an issue on the docs repo to add the redirect (please assign me or implement in /server.js)

Another implementation detail from that conversation:

> I should prepare some helper method, similar to `append_doc_link`